pdb/src/main/java/org/cbioportal/pdb_annotation/util/TextFile.java

```diff
@@ -0,0 +1,86 @@
+package org.cbioportal.pdb_annotation.util;
+import java.io.*;
+
+public class TextFile{
+
+	/**
+	 * Read in a given text file to a string
+	 * @param	file	file path of the desire input file
+	 * @return			the contents of the specified file
+	 */
+	public static String read(String file) throws IOException {
```

Comment:
For generic IO operations, it might be better to use an existing java library instead of manually implementing them. See Apache Commons for example (https://commons.apache.org/proper/commons-io/javadocs/api-release/org/apache/commons/io/FileUtils.html)